### PR TITLE
Sils/req

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

**Comment by abhsag24** [comment]
ack 15a1ed2 though there's a typo in the commit body makning --> making

File: requirements.txt [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
+cmakelint==1.*
+requests_mock==0.7.*
+vim-vint==0.3.*
 # Stable version 3.2 is problematic in Windows.
-# Use >=3.3 once nltk-3.3 is released.
-nltk~=3.1.0
-appdirs~=1.4.0
-pyyaml~=3.11
-coala_decorators~=0.2.1
+# Use >=3.3,==3.* once nltk-3.3 is released.
```
Comment (abhsag24) [inline requirements.txt]: wouldn't using ==3.* be problematic for windows if 3.2 is downloaded?

Comment (sils1297) [inline requirements.txt]: the comma is an and connection so that would be higher than 3.3

**Comment by abhsag24** [comment]
ack 0c23c6b

**Comment by sils1297** [comment]
ack 0c23c6b

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**